packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,377 @@
+## User Types API
+
+The User Types API provides operations to manage User Types. To manage the schemas associated with the User Types, refer to the [User Schema API reference](/docs/reference/api/schemas/#user-schema-operations) and see below for updates for this feature.
+
+### Get all User Types
```

Comment:
This will become just two`##`, and et cetera for the rest.

Reply:
Hm, actually, comparing this against the Users API and Schemas API pages, it looks like I should:
- change the top header "User Types API" to have a single #
- insert a new header "User Types Operations" with two ##
- leave the individual operations like "Get all User Types" with three ###
- change the "Request parameters" and "Response parameters" to have FIVE #####

Thoughts?

Reply:
Looks right, Don. Thank you.